master/internal/api_workspace.go

```diff
@@ -1292,7 +1300,8 @@ func getAutoGeneratedNamespaceName(ctx context.Context, wkspID int, tx *bun.Tx)
 		}
 		clusterID = ""
 	}
-	ns := generateNamespaceName(clusterID, wsToDelete.Name, wsToDelete.ID)
+
+	ns := generateNamespaceName(installationNamespace, clusterID, wsToDelete.Name, uint(wsToDelete.ID))
```

Comment:
```suggestion
	ns := generateNamespaceName(wsToDelete.Name, installationNamespace, clusterID, uint(wsToDelete.ID))
```
Per comment [here](https://github.com/determined-ai/determined/pull/9640/files#r1674391798)